jormungandr/src/network/p2p/comm/peer_map.rs

```diff
@@ -150,7 +150,6 @@ impl PeerMap {
         self.map
             .iter()
             .map(|(&id, data)| PeerInfo {
-                id,
```

Comment:
Notice that we also get the address in `id`. Maybe we should remove the address member from `PeerData`.